Documentation Update: Disabling Elasticsearch for an operation (#1131)
api-platform/docs: core/operations.md

```

That's it: your route is gone!

> ## Disabling ElasticSearch on Operations

Reviewer: Could you move this docs in `elasticsearch.md` instead please? It will be easier to discover.

Also, can you add a blank line after the title?

> After enabling [Elasticsearch Support](elasticsearch.md) in your project, there may come a time you would prefer not to read your data from your Elasticsearch server and would rather fetch your data directly from your database. To achieve this, you can pass in a parameter to your api operation with the value `elasticsearch: true`. A simple example is below.

Reviewer 2: ```suggestion
After enabling [Elasticsearch Support](elasticsearch.md) in your project, there may come a time you would prefer not to read your data from your Elasticsearch server and would rather fetch your data directly from your database. To achieve this, you can pass in a parameter to your api operation with the value `elasticsearch: true`. 
For example:
```
